modules/dnn/perf/perf_net.cpp

```diff
@@ -93,6 +93,41 @@ class DNNTestNetwork : public ::perf::TestBaseWithParam< tuple<Backend, Target>
     }
 };
 
+/*
+
+Tests are done with M1. data in millisecond (ms).
+
+| ViTs     | With Attention | Without Attention | ORT    |
+| -------- | -------------- | ----------------- | ------ |
+| vit_b_16 | 302.77         | 365.35            | 109.70 |
+| vit_b_32 | 89.92          | 116.22            | 30.36  |
+| vit_l_16 | 1593.32        | 1730.74           | 419.92 |
+| vit_l_32 | 468.11         | 577.41            | 134.12 |
+| VitTrack | 3.80           | 3.87              | 2.25   |
```

Comment:
Please remove the results from source code - it's enough to add to PR's description

Reply:
Running this models in `perf_net.cpp` is taking up too much time for now. How about we comment `PERF_TEST_P_(DNNTestNetwork, VIT_B_16)` and such for now until we get the close or even better inference speed as ORT? In this case, these perfamce results should be deleted and kept in the first comment of this PR.

Reply:
You may add a corresponding skip exception:

```cpp
applyTestTag(CV_TEST_TAG_LONG, CV_TEST_TAG_DEBUG_LONG);
```